code/modules/mob/mob_movement.dm

```diff
@@ -43,6 +43,14 @@
 	if(iscarbon(usr))
 		var/mob/living/carbon/C = usr
 		C.toggle_throw_mode()
+	else if(isrobot(usr))
+		var/mob/living/silicon/robot/R = usr
+		var/module = R.get_selected_module()
+		if(!module)
+			usr << "\red You have no module selected."

```

Comment:
"<span class='warning'>You have no module selected.</span>"

Wouldn't this be better?